### PR TITLE
Merge to Master: 1st home work

### PR DESCRIPTION
**Pull request by skrmnyi**

### CODE REVIEW

**Comment by skrmnyi** [comment]
next time I'll add your #nickname to the reviewers
just left it in the comments below, if it makes sense

File: src/main/java/org/example/MyClass.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    int year = 2023;
+
+    public static int sumMethod(int a, int b) {
+        int c = a + b;
+        return c;
+    }
+
+    public static void main(String[] args) {
+        sumMethod(1, 2);}
```
Comment (DarynaHorobei97) [inline src/main/java/org/example/MyClass.java:15]: Please format code (Ctrl+Alt+L)

File: src/main/java/org/example/MyClass.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+
+package org.example;
+
+public class MyClass {
+
+    int year = 2023;
+
+    public static int sumMethod(int a, int b) {
```
Comment (DarynaHorobei97) [inline src/main/java/org/example/MyClass.java:9]: It's better to name the method  - sumOfTwoNumbers

**Review by DarynaHorobei97 — COMMENTED** [review]
Good job!

**Pull request merged**